commit 09443b7960ae0f0a3ddcf56d1879e006c2790316 (refs/changes/93/2593/2)
Author: Bryan Cantrill <bryan@joyent.com>
Date:   2017-09-16T16:32:30+00:00 (2 years, 1 month ago)
    
    OS-6350 4K zfs_abd_chunk_size is inducing excessive ABD waste
    Reviewed by: Jason King <jason.king@joyent.com>
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Approved by: Jerry Jelinek <jerry.jelinek@joyent.com>

diff --git a/usr/src/uts/common/fs/zfs/abd.c b/usr/src/uts/common/fs/zfs/abd.c
index 4d7b2cb56a..92e7814df5 100644
--- a/usr/src/uts/common/fs/zfs/abd.c
+++ b/usr/src/uts/common/fs/zfs/abd.c
@@ -148,7 +148,7 @@ boolean_t zfs_abd_scatter_enabled = B_TRUE;
  * will cause the machine to panic if you change it and try to access the data
  * within a scattered ABD.
  */
-size_t zfs_abd_chunk_size = 4096;
+size_t zfs_abd_chunk_size = 1024;
 
 #ifdef _KERNEL
 extern vmem_t *zio_alloc_arena;
